<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighlightingsFrame</class>
 <widget class="QFrame" name="HighlightingsFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>16761562</width>
    <height>95</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLineEdit" name="patternEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="btnColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Delete All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="patternView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>src/ui/widgets/clickablelabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>HighlightingsFrame</receiver>
   <slot>addHighlighting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelete</sender>
   <signal>clicked()</signal>
   <receiver>HighlightingsFrame</receiver>
   <slot>deleteHighlighting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>HighlightingsFrame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patternEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>HighlightingsFrame</receiver>
   <slot>onTextEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patternEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>btnAdd</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDeleteAll</sender>
   <signal>clicked()</signal>
   <receiver>HighlightingsFrame</receiver>
   <slot>deleteAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patternView</sender>
   <signal>pressed(QModelIndex)</signal>
   <receiver>HighlightingsFrame</receiver>
   <slot>onSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>8388596</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>8388596</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnColor</sender>
   <signal>clicked()</signal>
   <receiver>HighlightingsFrame</receiver>
   <slot>changeColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16761190</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>8380780</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addHighlighting()</slot>
  <slot>deleteHighlighting()</slot>
  <slot>onTextEdited(QString)</slot>
  <slot>onSelectionChanged()</slot>
  <slot>changeColor()</slot>
  <slot>deleteAll()</slot>
 </slots>
</ui>
